### ISSUE TITLE
media query math issues

### ISSUE THREAD

**Issue by exigon** [leafo/scssphp#433]
This tricky thing left me wondering.

Notice how the compiler put out the width of the element wrong.

Code and expected output:
https://gist.github.com/exigon/656bc8576babc52462ae00ffb92cc18e

Output produced:
`@media screen and (min-width: 767px) {
    .element {
      float: left;
      display: block;
      margin-right: 2.35765%;
      width: 100%; 
    }
      .element:last-child {
        margin-right: 0; 
    } 
}`

Used plugins:
Bourbon 4.2.7
Neat 1.7.4

**Comment by ynamite** [comment]
Pretty big issue, for myself at least. Having exactly the same problem.

+1

**Comment by robocoder** [comment]
Considering this fixed as it's not reproduceable in v0.6.6. I get:
```
@charset "UTF-8";
html {
  box-sizing: border-box; }

*, *::after, *::before {
  box-sizing: inherit; }

@media screen and (min-width: 767px) {
    .element {
      float: left;
      display: block;
      margin-right: 2.35765%;
      width: 31.76157%; }
      .element:last-child {
        margin-right: 0; } }
```
